modules/foreman_job_template.py

```diff
@@ -405,7 +409,9 @@ def main():
             module.fail_json(msg="Cannot use '*' as a job template name!")
         # module params are priorized
         parsed_dict.update(entity_dict)
-        entity_dict = parsed_dict
+        # make sure certain values are set
+        entity_dict = template_defaults.copy()
```

Comment:
Kodos for safe programming! Even if it's rather unlilkely that the template_defaults be reused.